node/network/statement-distribution/src/lib.rs

```diff
@@ -2022,6 +2066,13 @@ impl metrics::Metrics for Metrics {
 				))?,
 				registry,
 			)?,
+			statements_out_view: prometheus::register(
+				prometheus::Counter::new(
+					"polkadot_parachain_statement_distribution_statements_out_view",
```

Comment:
Should we also differentiate between what kind of statements which arrived out of view via some label ("kind")?  `LargeStatement`,`Seconded` or `Valid`

Reply:
It is now only for tracking like `expected` out-of-view statements when we have received `Valid` but `Seconded` has been lost in transition. Probably, we might label this metric in future with different cases.